src/Compilers/CSharp/Portable/Errors/MessageID.cs

```diff
@@ -323,6 +324,7 @@ internal static LanguageVersion RequiredVersion(this MessageID feature)
             switch (feature)
             {
                 // C# preview features.
+                case MessageID.IDS_FeatureRecordStructs:
                 case MessageID.IDS_FeatureMixedDeclarationsAndExpressionsInDeconstruction:
                     return LanguageVersion.Preview;
```

Comment:
There is another part of this switch that returns LanguageVersion.Preview:

https://github.com/dotnet/roslyn/blob/f3849782b678e611f598bb51d57b6dd93471b6a9/src/Compilers/CSharp/Portable/Errors/MessageID.cs#L357-L359

Better to group them together?

Reply:
Yes, we'll want to sort this at some point but I think doing it here/now would be worse. We'll have an opportunity to sort this list when moving features from preview to C# 10. Doing it now would just create conflicts between feature branches.